src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs

```diff
@@ -59,6 +59,12 @@ public interface IInteractsWithApps : IExecuteMethod
         /// <param name="appPath">a string containing the id of the app.</param>
         void RemoveApp(string appId);
 
+        /// <summary>
+        /// Activates an App.
```

Comment:
Activates the given app by moving to foreground if it is running in the background or starting it if it is not running yet

Reply:
An error will be thrown if the app with the given identifier is not present on the device under test